test/e2e/testdata/dependencies-continueon-dag-workflow.yaml

```
@@ -0,0 +1,33 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: dependencies-continueon-dag-workflow
```

```suggestion
  name: dependencies-continueon-failure-workflow
```

I would specify that it's a Workflow that fails
DAG is redundant as `dependencies` are only for DAGs